### ISSUE TITLE
"Align wrapped list items to the indent level" malfunctions sometimes

### ISSUE THREAD

**Issue by joochlee** [CalebJohn/joplin-rich-markdown#50]
Hi, 

First of all, I thank you for super useful plugin! 👍

I would like to report a bug regarding "align wrapped list items". 

It often malfunctions (usually occur when I restart joplin client app), please see below image.
![image](https://user-images.githubusercontent.com/16387632/212245319-ea047fce-0334-420e-b559-3e0ad30727bb.png)

This symptom is gone after I disable `Align wrapped list items to the indent level` and enable it again. 
![image](https://user-images.githubusercontent.com/16387632/212245683-134e160e-42ca-47bd-ac18-5793fbc8c68b.png)

**Comment by yajo** [comment]
The feature is actually not working at all AFAICS.

**Comment by ernstki** [comment]
Agreed, it seems to be non-functional with the latest (v0.16.0) and previous versions of the plugin, with a Joplin app that's only one point release behind the current.

![A screenshot of a recipe, with the Rich Markdown plugin, showing several bulleted lists that used to wrap (under the bullet marker), but don't seem to properly do so anymore with the last few releases of the plugin](https://github.com/user-attachments/assets/ff92639a-5308-4f30-8ef0-4d7b51143a68)

**Comment by CalebJohn** [comment]
Unfortunately that feature is not working yet in the new version of the Joplin markdown editor. You can switch back to the old editor to get aligned lists. I will get it working for the new editor, but it was always a hack and I just have to figure out a new hack.

**Comment by ernstki** [comment]
> You can switch back to the old editor to get aligned lists.

Aha, Indeed it does! I saw stuff in the commit log about CodeMirror 6 and wondered about this. So far the small number of plugins I use seem to be OK with it, in spite of the warning in the UI about enabling the legacy editor.

![Screenshot of "Use the legacy Markdown editor" in Joplin's General → Show Advanced Settings section](https://github.com/user-attachments/assets/180cd780-51dd-4d18-a045-e3265851cefb)

Whatever you did the first time, it was a clever hack! Makes working with deeply nested lists a lot more ergonomic, and your efforts there are totally appreciated.
